pkg/sql/pgwire/testdata/pgtest/decimal

```diff
@@ -35,3 +35,36 @@ ReadyForQuery
 {"Type":"DataRow","Values":[{"text":"0"}]}
 {"Type":"CommandComplete","CommandTag":"SELECT 1"}
 {"Type":"ReadyForQuery","TxStatus":"I"}
+
+send
+Parse {"Name": "s1", "Query": "SELECT 10000::decimal, $1::decimal"}
+Bind {"DestinationPortal": "p1", "PreparedStatement": "s1", "ParameterFormatCodes": [1], "Parameters": [[0, 1, 0, 1, 0, 0, 0, 0, 0, 1]]}
```

Comment:
can you also add a test that sends an extra `0000` at the end? (see my second bullet point for https://github.com/cockroachdb/cockroach/issues/56907#issuecomment-733194561)

Reply:
i may not understand -- where should i add the `0000`?

> When CockroachDB decodes the value NPGSQL provides, I think we will return 10000 instead of 1E+4 if you make add extra 00 00 at the end (other drivers do this so perhaps this is the problem). I think 10000 0000 is broken too, etc. for any value with 0000 as the last 4 digits. 

Meaning add `00 00` to the end of the binary parameter passed in?

Reply:
Yeah sorry I mean if we do introduce the extra 0000 to read

Reply:
hm yeah if i add `0000` to the end of the binary param, it is rejected as invalid binary data